terraform/aws/minikube/main.tf

```diff
@@ -10,6 +10,12 @@ resource "aws_default_subnet" "default" {
   }
 }
 
+
+data "aws_subnet_ids" "vpc_subnets" {
+  vpc_id=var.vpc_id
+  count=var.vpc_id != "" ? 1 : 0
```

Comment:
Counts ALWAYS should be at first line and separated by empty line

Also, please, use spaces

Like this:
```terraform
data "aws_subnet_ids" "vpc_subnets" {
  count = var.vpc_id != "" ? 1 : 0

  vpc_id = var.vpc_id
}
```

Reply:
+